src/plugins/vuetify.ts

```diff
@@ -0,0 +1,31 @@
+// Styles
+import '@mdi/font/css/materialdesignicons.css'
+import 'vuetify/styles'
+import 'vuetify/dist/vuetify.min.css'
+import '@mdi/font/css/materialdesignicons.min.css' // ensure you are using css-loader
+import '@/assets/styles/base.scss'
+import '@/assets/styles/layout.scss'
+import '@/assets/styles/overrides.scss'
+import * as components from 'vuetify/components'
+import * as directives from 'vuetify/directives'
+// Vuetify
+import { createVuetify } from 'vuetify'
+
+export default createVuetify({
+  theme: {
+    defaultTheme: 'bcgov',
+    themes: {
+      bcgov: {
+        colors: {
+          primary: '#1669bb', // same as $$primary-blue
+          darkBlue: '#38598a',
+          error: '#d3272c',
+          success: '#1a9031',
+          warning: '#ffc107'
+        }
```

Comment:
Total aside, but maybe we should add that dark green here and simplify the components.

Reply:
Sev, so here's the value.
![green-darken-2](https://github.com/bcgov/business-create-ui/assets/122301442/0b278298-af51-4deb-a66f-ad4717a5a3ec)

You want me to add the value here and in the components reference the color? I think using the green-darken-2 color in the components is OK since it's a Vuetify color so we can straight forward use it. Thoughts?

Reply:
I think either is equivalent, but using our own named colour is less prone to typos. Eg, "darkGreen2".

Reply:
Up to you if you do it. What would I do? 😁

Reply:
Ohh, I see. I'm on it 🙌

Reply:
(I only mentioned it because I saw many instances of it.)

Reply:
You fixed it. Resolving.